saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -277,6 +266,17 @@ def perform_mutation(cls, _root, info, order, **data):
             errors = prepare_insufficient_stock_order_validation_errors(exc)
             raise ValidationError({"stocks": errors})
 
+        if approved:
```

Comment:
What do you think about moving this to saleor/order/actions.py file. There is a method `saleor.order.actions.order_fulfilled` Which is triggered by this mutation. It contains actions like this one so I would consider moving this action there.